packages/terra-clinical-detail-view/src/terra-dev-site/doc/clinical-detail-view/AccessibilityGuide.4.doc.mdx

```diff
@@ -0,0 +1,161 @@
+
+It is highly recommended to utilize the `level` prop to set the appropriate heading level of the Title to match the hierarchical structure of the page. The level prop will also set the SecondaryTitles and the title of the Detail List structures to the appropriate heading level to ensure a hierarchical content structure.
+
+Code Example:
+```jsx
+const item1 = (<LabelValueView label="Start Date" textValue="12/12/12 00:00AM" isChildOfDescriptionList />);
+
+<DetailView
+  title="Multiple Ingredients"
+  level={4}
+  secondaryTitles={['M. V. I. Adult 10 mL = 10 mL', 'Sodium bicarbonate 8.4% 50 mL', 'Dextrose 5% in Water 1000 mL']}
+  details={[
+    (
+      <DetailView.DetailList title={detail1title} key="Indicators">
+        <DetailView.DetailListItem item={detail1item1} />
+      </DetailView.DetailList>
+    ),
+  ]}
+/>
+```
+
+#### Named Section Behavior:
+
+All of the content related to the Detail View should be understood as a section and the `title` prop is used to programmatically create a label for the region.  It is critical to screen reader users that the title prop be set to a value that best represents the region that groups the details in the section within the page. Screen reader users rely on the title to know the beginning and end of a region, and to navigate to the specific region on the page. Leaving the title off of the section hurts the accessibility of the component by removing the programmatic indication of the region on the page. The region/section won't be announced by Assistive Technology (AT) if there is no programmatically associated label. 
+
+#### Accessibility Considerations: Caption for graphs:
+
+The graph prop is used to set visualization content such as graphs, or images based on the data. This graph prop is wrapped in an [`<figure>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure) element. 
+Figure captions help the users understand and correctly interpret the visualization. 
+It is critical to screen reader users that the caption be set with [`<figcaption>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption) element as the first, or the last element to provide the caption for the figure.
+
+```
+Note: figcaption content is not a replacement for the alt text that needs to be supplied for images in the figure element.
+```
+
+When using figcaption element, which requires a parent element, it is necessary to wrap the elements within a `React.Fragment` block.
+
+**Example:**
+
+<DetailViewDivided />
```

Comment:
@scottwilmarth @brooksienoodlesoup - I am not sure how we need the examples. Is it just the code snippets like I have [here](https://github.com/cerner/terra-clinical/pull/878/files#diff-d2a39e273f246a126dc732d4ce37a6cb48c4b643713ebd96e1544b4c7da50202R42-R58) or do we need to create one as we have on the Implementation guide? 
This is how it would look on the page and we can hit the code button on the bottom of the example to look at the code.
![Screen Shot 2023-06-27 at 8 26 15 PM](https://github.com/cerner/terra-clinical/assets/55503680/8cd07e3e-05e4-4ef0-9108-fe471eafe9d5)

Let me know how we need the examples. I can update the other examples as well.

Reply:
# Ideas for sample figure and figcaption content
Hi @vinaybhargavar and @scottwilmarth  - My thought is that it would be best to use a .jpg or .gif image (not svg), in the example figure.  We would want to provide alt text for that image.  Maybe the alt text for the image is short and describes what's on the face of the image contained in the figure.  Then, we would put text in the figcaption that provides a longer explanation of the larger purpose of the image in the figure on the page.  All of this content may need to come from clinical content folks.  Hoping Scott can help guide us on this point.  

I'm finding it hard to find a perfect figure/figcaption example quickly, but [here's an example of a figure we can work with for explanation purposes](https://html.com/tags/figcaption/).  The alt text for the flamingo jpg is simply "flamingo" whereas the figcaption text for this figure could be something like "As a flamingo dines on algae and brine shrimp, its body metabolizes the pigments — turning its feathers pink."  Does that make sense?  I'm happy to provide further explanation if that will help.

Reply:
@brooksienoodlesoup - Can you let me know which image you are referring to for the figure element? I believe now we are building a visualization (graph) for the figure. Or do you mean using the flamingo example instead of the graph?

Reply:
#figure and figcaption assistive technology support

At the present, assistive technology support is weak for the figure and figcaption elements. For example, the semantic value of the figure or figcaption elements is not passed along to users in either Chrome+VO or Edge+Jaws.  However, image and alt text content is announced, as is the content of the figcaption element via regular reading mode screen reader browsing of the content. 

[There is a known "bug" in Jaws, where content within the figure element is announced along with the phrase "has details."](https://github.com/FreedomScientific/VFO-standards-support/issues/657)

[Here is a resource that offers a bit more detail on assistive technology support for figure and figcaption](https://a11ysupport.io/tech/html/figcaption_element#related-tests).

Reply:
@vinaybhargavar I don't mean to use the Flamingo image for our example.  By all means, we should use an image with clinical relevance.  I just shared the flamingo example as a way of showing how a figure element will hold an image with a short alt text phrase, and the figcaption will usually hold a longer text description with extra context of why the image is important.  The flamingo image and text should not be used in our actual documentation.

Reply:
Thanks @brooksienoodlesoup, that makes sense. I can add the alt text and update the figcaption with the content from either the clinical content folks or @scottwilmarth.

Reply:
Hi @vinaybhargavar Please replace the flamingo with the svg chart.  Thank you.

Reply:
Discussed with @brooksienoodlesoup. Since it can take some time to get a clinical image with an explanation we decided to revert to the previous example of having a graph with figcaption here.
The graph is an <svg>, which has not been made accessible. I will log a Jira to figure out how to make the <svg> accessible as a future story to the backlog.

Reply:
Logged a jira for make `<svg>` accessible - https://jira2.cerner.com/browse/UXPLATFORM-9289